src/content/guides/minimal-startup.md

```diff
@@ -0,0 +1,389 @@
+
+The simpliest webpack command in the current version of webpack is "webpack <entryFilename.js>". Before issuing this command the <entryFilename.js> file must be created. This file will be named entryMain.js in this case.
+
+__webpack-basics/entryMain.js__
+``` JavaScript
+  console.log('Console.log message inside entryMain.js')
+```
+
+Now we can webpack this file.
+
+``` bash
+> webpack entryMain.js
+
+Hash: ae8ebe3170f4c6373946
+Version: webpack 4.17.1
+Time: 1660ms
+Built at: 2018-08-25 11:31:33
+  Asset       Size  Chunks             Chunk Names
+main.js  984 bytes       0  [emitted]  main
+Entrypoint main = main.js
+[0] ./entryMain.js 56 bytes {0} [built]
+
+WARNING in configuration
+The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
+You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/concepts/mode/
+```
+
+Congratulations, you have webpacked your first file!!!
+
+#### Confirm that webpack worked.
+
+First ignore the warning about the mode option for now. Next confirm the following new directory structure.
+
+__project__
+``` diff
+  webpack-basics
++ |- dist
++   |- main.js
+  |- entryMain.js
+```
+
+Note that a dist folder has been created inside your project folder. Change into it and confirm that a main.js file has been created. The dist folder is the default location for the bundled file and webpack created it automatically since it did not exist. Similarly webpack "bundled: the entryMain.js file into a main.js file.  This can be seen in the output line: Entrypoint main = main.js.
+
+Open the bundled main.js file and observe that it consists of JavaScript code without line breaks. Much of the code will be unfamiliar, but you should be able to find the content of your entryMain.js file inside the main.js file at the end of the file.
+
+__webpack-basics/dist/main.js__
+``` JavaScript
+!function(e){var n={};function t(r){if(n[r])return n[r].exports;var o=n[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,t),o.l=!0,o.exports}t.m=e,t.c=n,t.d=function(e,n,r){t.o(e,n)||Object.defineProperty(e,n,{enumerable:!0,get:r})},t.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},t.t=function(e,n){if(1&n&&(e=t(e)),8&n)return e;if(4&n&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(t.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&n&&"string"!=typeof e)for(var o in e)t.d(r,o,function(n){return e[n]}.bind(null,o));return r},t.n=function(e){var n=e&&e.__esModule?function(){return e.default}:function(){return e};return t.d(n,"a",n),n},t.o=function(e,n){return Object.prototype.hasOwnProperty.call(e,n)},t.p="",t(t.s=0)}([function(e,n){console.log("Console.log message inside entryMain.js")}]);
```

Comment:
webpack's raw output is a subject to change with dep or and wrbpack's updates, this is not really maintainable or useful snippet